tests/unit/python/fledge/services/core/api/test_certificate_store.py

```diff
@@ -300,24 +198,18 @@ async def test_bad_type_delete_cert(self, client):
             assert 1 == patch_cfg.call_count
 
     @pytest.mark.parametrize("cert_name, param", [
-        ('fledge.cert', '?type=cert'),
         ('fledge.json', '?type=cert'),
         ('fledge.pem', '?type=cert'),
-        ('fledge.key', '?type=key'),
         ('test.cer', '?type=cert'),
```

Comment:
would it not it better to keep .key .cert check as well? may with with name foo.key foo.cert

Reply:
In that case we need to add new test data files